types/auth0/index.d.ts

```diff
@@ -509,13 +509,64 @@ export interface StatsParams {
   to: string;
 }
 
+export type Job = ImportUsersJob | ExportUsersJob | VerificationEmailJob;
+
+export type JobFormat = 'csv' | 'json';
+
+export type JobStatus = 'pending' | 'processing' | 'completed' | 'failed';
+
+export interface ExportUsersJob {
+    id: string;
+    type: 'users_export';
+    status: JobStatus;
+    created_at?: string;
+    connection_id?: string;
+    fields?: ExportUserField[];
+    location?: string;
+    format?: JobFormat;
+}
+
+export interface ImportUsersJob {
+    id: string;
+    type: 'users_import';
+    status: JobStatus;
+    created_at?: string;
+    connection_id?: string;
+    upsert?: boolean;
+    external_id?: string;
+    send_completion_email?: boolean;
+}
+
+export interface VerificationEmailJob {
+    id: string;
+    type: 'verification_email';
+    status: JobStatus;
+    created_at?: string;
+}
+
 export interface ImportUsersOptions {
-  connection_id: string;
-  users: string;
+    users: string;
+    connection_id: string;
+    upsert?: boolean;
+    external_id?: string;
+    send_completion_email?: boolean;
 }
 
```

Comment:
Changing the interface name from `UserIdParams` to `VerificationEmailOptions` will break users using this type. I don't think that's an intended behavior. I suggest changing back the name unless you feel this is a different interface that needs to be fixed.

Reply:
The name change has been reverted